assets/app/view/game/buy_companies.rb

```diff
@@ -28,7 +28,7 @@ def render_companies
           },
         }
 
-        companies = @game.purchasable_companies.sort_by do |company|
+        companies = @game.purchasable_companies(@game.current_entity).sort_by do |company|
```

Comment:
why is this changed? current_entity should be the default

Reply:
My bad, then. But everywhere else where it's called, it is passing then entity

Reply:
then that should be refactored as it's redundant

Reply:
I'll take a look to see which ones are redundant and open a separate PR removing them if possible. (this one is already re-removed)